src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -2920,6 +2921,20 @@ function doSpeedWalk()
     end
 end
 
+function map.pauseSpeedwalk()
```

Comment:
looks good - can you also add paused and resumed events? we added them 3 days ago for other functions, see [[<
![image](https://user-images.githubusercontent.com/110988/141255384-3b13ba08-2eb2-4c7f-9c5d-f8191f728528.png)
](https://github.com/Mudlet/Mudlet/pull/5542/files#diff-91354c6538a2f14aa1ec932dca871f88d3ba833d0e4e78e3690bf4677da47e94R443)](https://github.com/Mudlet/Mudlet/pull/5542/files#diff-91354c6538a2f14aa1ec932dca871f88d3ba833d0e4e78e3690bf4677da47e94R443)

Reply:
ok